domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs

```diff
@@ -81,6 +82,18 @@ public async Task SeedData(IEnumerable<TransferAgreement> transferAgreements)
         }
     }
 
+    public async Task SeedConnections(IEnumerable<Connection> connections)
+    {
+        using var scope = Services.CreateScope();
+        var dbContext = scope.ServiceProvider.GetRequiredService<ApplicationDbContext>();
+        await TruncateTables(dbContext);
```

Comment:
It is a bit funky for me, that when seeding connections, then tables are truncated for Transfer Agreements.

Reply:
Made separate truncate for connections only